client/src/components/RIC-Widget/ProductCard.jsx

```diff
@@ -48,9 +52,9 @@ const ProductCard = ({ productId, index, listState, triggerDelete, triggerModal
       <img src={image} alt={name} className='ProductImage'></img>
       <div>{category}</div>
       <b>{name}</b>
-        {isSale
-          ? <div>${salePrice} <strike>${price}</strike></div>
-          : <div>${price}</div>
+        {price.salePrice
+          ? <div><span id="salePrice">${price.salePrice}</span> <strike>${price.default}</strike></div>
+          : <div>${price.default}</div>
```

Comment:
Don't think it's necessary to have both a div and a span here.

Reply:
I have to have it wrapped in a div, otherwise webpack yells at me.